feat: add rippled API v2 support and use as default (#720)
XRPLF/xrpl-py: CHANGELOG.md

## [[Unreleased]]
- Add support for the DeliverMax field in Payment transactions

### Added

> - Update all requests to use rippled API v2 responses

Reviewer: ```suggestion
- Use rippled API v2 as default in requests
```

### Added
- Add support for the DeliverMax field in Payment transactions

> - Use rippled API v2 as default in requests

Reviewer 2: I think this should be at the top (or highlighted some how) since this is a major change. Other than that, LGTM.

Author: fixed in 4179077
